account-kit/react/src/components/auth/card/loading/oauth.tsx

```diff
@@ -25,6 +27,16 @@ export const CompletingOAuth = () => {
     }
   }, [isConnected, oauthWasCancelled, setAuthStep]);
 
+  useEffect(() => {
+    const cancel = signer?.on("emailAuthLinkingRequired", (email) => {
+      setAuthStep({ type: "otp_verify", email });
```

Comment:
do we need to show different copy to the otp_verify screen explaining why folks are getting an OTP code during oauth login?

Reply:
I considered it but I also thought that it won't register as unusual at all for a user to be prompted with an OTP code here so it's not worth calling attention to it. Curious what other people think.